ambry-api/src/main/java/com.github.ambry/router/RouterErrorCode.java

```diff
@@ -82,7 +82,11 @@
   /**
    * TTL of Blob has expired and so Blob cannot be retrieved.
    */
-  BlobExpired;
+  BlobExpired,
+  /**
+   * The range offsets provided for a getBlob operation are invalid for the specified blob.
+   */
+  RangeNotSatisfiable;
```

Comment:
This comment is for line 96 and unrelated to this PR. Could you rebase and remove `convertCoordinatorErrorToRouterErrorCode()`? I believe it should not have any more usages.